core/src/main/scala/org/apache/spark/scheduler/MapStatus.scala

```diff
@@ -193,8 +216,21 @@ private[spark] object HighlyCompressedMapStatus {
     } else {
       0
     }
+    val threshold =
+      SparkEnv.get.conf.getLong("spark.shuffle.accurateBlkThreshold", 100 * 1024 * 1024) max
+        SparkEnv.get.conf.getInt("spark.shuffle.accurateBlkThresholdByTimesAvg", 2) * avgSize
```

Comment:
Agree about moving it to `org.apache.spark.internal.config`.

I am fine with it either way - a constant size does have the potential for degenerating into `CompressedMapStatus` for large shuffle's.
Btw, can we explicitly use math.max() ? instead of `v1 max v2` ? I thought that was the idiom used in spark ?